### DISCUSSION TITLE
KLayout not showing the layout live

### DISCUSSION THREAD

**Discussion by vangapandu95** [gdsfactory/gdsfactory#1262] [Q&A] [answered]
I am trying to run an example bezier curve python file in the components folder of the gdsfactory directory using VScode. The code rns without error. But the layout/component doesnt show up in the KLayout. I recall that it did show up once when I ran the python code in the VScode window. 
I have the packages installed in the KLayout for the GDSFactory. What could be the reason the KLive is not working properly?
![Installed packages](https://user-images.githubusercontent.com/54287784/217969876-adf1013f-7d0a-4061-b760-f7f00ddbfa1a.PNG)
![Vscode KLayout interface](https://user-images.githubusercontent.com/54287784/217969888-a64471e3-f91c-43a0-b557-cf37200611d2.PNG)

**Comment by joamatab** [comment, marked as answer]
Hi

I think updating to the latest gdsfactory fixed your issue

**Comment by vangapandu95** [comment]
Yes it did. Thanks!
